app/assets/javascripts/controllers/ansible_credentials/ansible_credentials_form_controller.js

```diff
@@ -40,6 +40,10 @@ ManageIQ.angular.app.controller('ansibleCredentialsFormController', ['$window',
       vm.modelCopy = angular.copy( vm.credentialModel );
       miqService.sparkleOff();
     }
+
+    $q.all([optionsPromise, dataPromise])
+      .then(retrievedCredentialDetails)
+      .catch(miqService.handleFailure);
```

Comment:
Sorry, this line can't be here, otherwise, on error, you'll see the error twice.

We could theoretically keep it and remove those 2 `catch` above, but then you'd only see the first error if both fail.

But assuming `retrievedCredentialDetails` doesn't throw, we can just remove this line..